doc/tutorials/using_grub.rst

```diff
@@ -0,0 +1,188 @@
+.. _using_grub:
+
+Using GRUB to boot ACRN
+#######################
+`GRUB <http://www.gnu.org/software/grub/>`_ is a multiboot boot loader
+used for many popular Linux distributions. It also supports booting the
+ACRN hypervisor.
+See `<http://www.gnu.org/software/grub/grub-download.html>`_
+to get the latest GRUB source code and `<https://www.gnu.org/software/grub/grub-documentation.html>`_
+for detailed documentation.
+
+The ACRN hypervisor can boot from `multiboot protocol <http://www.gnu.org/software/grub/manual/multiboot/multiboot.html>`_
+or `multiboot2 protocol <http://www.gnu.org/software/grub/manual/multiboot2/multiboot.html>`_.
+Comparing with multiboot protocol, the multiboot2 protocol adds UEFI support.
+
+The multiboot protocol is supported by the ACRN hypervisor by default.
```

Comment:
I was struggling a bit to follow these couple of paragraphs. I think it would be better to explain that both protocols are supported by Grub and are distinguished by the use of different commands put in the Grub configuration files.